app/src/main/java/com/example/triptracker/viewmodel/RecordViewModel.kt

```diff
@@ -38,20 +43,19 @@ class RecordViewModel : ViewModel() {
   val title = mutableStateOf("")
 
   // Private mutable list of LatLng points
-  private var _latLongList = mutableListOf<LatLng>()
+  // private var _latLongList = mutableListOf<LatLng>()
   // Public immutable list of LatLng points
   val latLongList: List<LatLng>
     get() = _latLongList
 
   // Private mutable list of Pin points
-  private var _pinList = mutableListOf<Pin>()
+  // private var _pinList = mutableListOf<Pin>()
 
   // Public immutable list of LatLng points
   val pinList: List<Pin>
     get() = _pinList
 
   // Geocoder object to interact with the Nominatim API
-  private val geocoder = NominatimApi()
```

Comment:
why is the geocoder, the latlng list and the pin list removed ?

Reply:
if you look at the head of the function it's initialized here. otherwise the test won't pass